### DISCUSSION TITLE
Rematch/Char select/Quit menu for versus modes.

### DISCUSSION THREAD

**Discussion by UrsoManso** [ikemen-engine/Ikemen-GO#1454] [Feature Request]
### Is your feature request related to a problem? Please describe.

Plenty of times when I'm playing quick match I wish I could straight up rematch instead of going back to char select and repeat the process.

### Describe the solution you'd like

Once the victory screen time ends a menu would pop up with the options to rematch, go back to char select or quit to main menu. Many comercial games have this handy feature so I thought Ikemen could make use of it as well.

**Comment by Al-fata-tayeb** [comment]
F4 ?

**Comment by UrsoManso** [comment]
F4 is a debug feature. I'm talking about an actual screenpack feature.

**Comment by K4thos** [comment]
this is already implemented. Quote from data/system.base.def
```
	menu.itemname.back = "Continue"
	menu.itemname.menuinput = "Button Config"
	menu.itemname.menuinput.keyboard = "Key Config"
	menu.itemname.menuinput.gamepad = "Joystick Config"
	menu.itemname.menuinput.empty = ""
	menu.itemname.menuinput.inputdefault = "Default"
	menu.itemname.menuinput.back = "Back"
	menu.itemname.commandlist = "Command List"
	menu.itemname.characterchange = "Character Change"
	menu.itemname.exit = "Exit"

	; Unassigned menu items:
	;menu.itemname.reset = "Round Reset"
	;menu.itemname.reload = "Rematch"	
```

**Reply by UrsoManso** [reply]
But this is for the in-game options menu right?! Because if that's the case then there's nothing to do with my request.

**Comment by imedox** [comment]
what about P2 join game in char selection with P1 in Arcade Mod !!! it can't at the last build.
